test/unit/test_crs.cpp

```diff
@@ -6551,10 +6551,19 @@ TEST(crs, crs_alterCSLinearUnit) {
     }
 
     {
-        // Not implemented on compoundCRS
         auto crs =
             createCompoundCRS()->alterCSLinearUnit(UnitOfMeasure("my unit", 2));
-        EXPECT_TRUE(createCompoundCRS()->isEquivalentTo(crs.get()));
+        auto compoundCRS = dynamic_cast<CompoundCRS *>(crs.get());
```

Comment:
for the sake of completness, and test line 393 of crs.cpp, we'd need a test on a type of CRS that has no implementation of alterCSLinearUnit(). For example a temporal or parametric CRS

Reply:
done